torchvision/csrc/models/vgg.cpp

```diff
@@ -32,7 +32,7 @@ torch::nn::Sequential makeLayers(
 void VGGImpl::_initialize_weights() {
   for (auto& module : modules(/*include_self=*/false)) {
     if (auto M = dynamic_cast<torch::nn::Conv2dImpl*>(module.get())) {
-      torch::nn::init::xavier_normal_(M->weight); // TODO kaiming
+      torch::nn::init::kaiming_normal_(M->weight);
```

Comment:
Question: is the default behavior of `kaiming_normal_` in C++ the same as in Python?
Because in Python we pass `fan_out` and `relu`
https://github.com/pytorch/vision/blob/f75b8148e03cc3155aff6431cd785ea02f299420/torchvision/models/vgg.py#L51

Reply:
I forgot. Fixed it.